db/migrate/20230711150925_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
- [ ] Please include a column for storing the user's photo in the `users` table, specifying the appropriate data type and any necessary constraints. ``t.string :photo`` :pray:

Reply:
I have implemented this requirement. 
You didn't check the other migrations. 

Please confirm this from the schema.rb file.

Check this migration

https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/migrate/20230711165705_add_photo_to_users.rb#L1C53-L1C53

This is the schema.rb file

https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/schema.rb#L1C53-L1C53